localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -559,18 +545,21 @@ geometry_msgs::msg::PoseWithCovarianceStamped NDTScanMatcher::align_using_monte_
 
   for (unsigned int i = 0; i < initial_poses.size(); i++) {
     const auto & initial_pose = initial_poses[i];
-    const NdtResult ndt_result = align(initial_pose);
+    const Eigen::Matrix4f initial_pose_matrix = pose_to_matrix4f(initial_pose);
+    auto output_cloud = std::make_shared<pcl::PointCloud<PointSource>>();
```

Comment:
`output_cloud` is defined in line 544, too.  
So, please remove the `output_cloud` defined in the for-loop.

Reply:
Thanks! Resolved here: https://github.com/autowarefoundation/autoware.universe/pull/2199/commits/6580214b98dbe68c20c7347b1fe5434cba74fc09